util/testkit/testkit.go

```diff
@@ -48,6 +48,27 @@ func (res *Result) Check(expected [][]interface{}) {
 	res.c.Assert(got, check.Equals, need, res.comment)
 }
 
+// CheckAt asserts the result of selected columns equals the expected results.
+func (res *Result) CheckAt(cols []int, expected [][]interface{}) {
+	for _, e := range expected {
+		if len(cols) != len(e) {
+			panic("lens of cols and expected cols should be equal")
```

Comment:
why use panic here but not res.c.Assert ?

Reply:
fixed